### PR TITLE
Cogscarabs now benefit from hat armor if Ratvar is alive.

### PR DESCRIPTION
**Pull request by ChangelingRain**
Get that clockwork armor, scarab.

### CODE REVIEW

**Comment by coiax** [comment]
Why can't drones just always benefit from hat armor

**Comment by ChangelingRain** [comment]
powergaming drones.

**Comment by Armhulen** [comment]
Is it power gaming though? The cogscarab needs to work with teammates to get armor, how horrible.

**Comment by ChangelingRain** [comment]
All its teammates need to do is cast Fellowship Armory, which will give the scarab, and everyone else, armor.

**Comment by Armhulen** [comment]
Cogscarab working with teammates and using resources to get armor = Powergaming

Cultist working with teammates and using resources to get armor = Not powergaming

I don't get it

**Comment by ChangelingRain** [comment]
coiax's question was about drones specifically, not cogscarabs.
drones would absolutely grab armor to fuck with people

**Comment by PKPenguin321** [comment]
Plus combatscarabs are shit

**Comment by Armhulen** [comment]
Combatscarabs are shit

at combat, they're terrible for fighting and they move slower and are hit by projectiles. drones who grab it for armor are shit too but they can grab like any other armor hat so who cares if they want a cool hat

File: code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm [hunk 1]
```diff
@@ -166,9 +166,15 @@
 		..()
 
 /mob/living/simple_animal/drone/cogscarab/can_use_guns(obj/item/weapon/gun/G)
-	changeNext_move(CLICK_CD_RANGE*4) //about as much delay as an unupgraded kinetic accelerator
+	if(!ratvar_awakens)
+		changeNext_move(CLICK_CD_RANGE*4) //about as much delay as an unupgraded kinetic accelerator
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: brap brap I'm the drone with the gat

**Pull request merged**